CMakeCache.txt

```diff
@@ -0,0 +1,356 @@
+# This is the CMakeCache file.
+# For build in directory: /home/liyan/vml/OpenVML
```

Comment:
It certainly is generated file and we will clean it. Many thanks.

Yan Li
From my phone.

> 在 2015年8月14日，12:18，Xin Huang <notifications@github.com> 写道：
> 
> In CMakeCache.txt:
> 
> > @@ -0,0 +1,356 @@
> > +# This is the CMakeCache file.
> > +# For build in directory: /home/liyan/vml/OpenVML
> Isn't this file a CMake cache file and will be regenerated every time during configuration? Maybe it's a mistake to add these files.
> 
> ―
> Reply to this email directly or view it on GitHub.
>